contracts/optimistic-ethereum/OVM/execution/OVM_StateManager.sol

```diff
@@ -392,7 +392,35 @@ contract OVM_StateManager is iOVM_StateManager {
     {
         return totalUncommittedAccounts;
     }
-    
+
+    function wasAccountChanged(
```

Comment:
Note this function will return `false` if the `ITEM_STATE` is `ItemState.ITEM_COMMITTED`, not sure if intended behavior but would update either name or implementation to reflect.

Reply:
Great point, will change.